src/main.ts

```diff
@@ -61,25 +89,37 @@ export async function activate(context: ExtensionContext) {
 		);
 	}
 
-	const command = await getServerPath(context, outputChannel);
+	let server = await getServerPath(context, outputChannel);
 
-	if (!command) {
-		await window.showErrorMessage(
-			"The Biome extensions doesn't ship with prebuilt binaries for your platform yet. " +
-				"You can still use it by cloning the biomejs/biome repo from GitHub to build the LSP " +
-				"yourself and use it with this extension with the biome.lspBin setting",
+	if (!server.command) {
+		const action = await window.showWarningMessage(
+			"Could not find Biome in your dependencies. Either add the @biomejs/biome package to your dependencies, or download the Biome CLI.",
```

Comment:
CLI is misleading, it should be called binary, I think

Reply:
Fair enough